Updating docs and comments (#57)
bloomberg/bbit-learning-labs: Tech-Lab-On-Campus/NewsFeed/docs/index.md

✨ You should now be ready to develop! ✨

#### Issues with Docker 

> If you are encountering issues with docker and cannot find the root cause, you can use codespaces on Github:

Reviewer: Maybe you can just leave it at this
```suggestion
If you are encountering issues with docker, you can use codespaces on Github instead:
```

Author: Changed this

1. Fork the repo 
2. Navigate to the forked repo 
3. Click on the green `< > Code` button 
4. Select the Codespaces tab and click on the `+` button 

> 5. cd into `Tech-Lab-On-Campus/NewsFeed/`

Reviewer: ```suggestion
5. In the codespace, open up a new terminal and run `cd Tech-Lab-On-Campus/NewsFeed/`
```

Author: Changed
